File: SysAudit-VB/Attributes/OrderAttribute.vb
<AttributeUsage(AttributeTargets.Property, Inherited:=False, AllowMultiple:=False)>
Public NotInheritable Class OrderAttribute : Inherits Attribute

    Public ReadOnly Property Order As Integer

    Public Sub New(Optional Order As Integer = 0)
        Me.Order = Order
    End Sub

End Class

File: SysAudit/SysAudit/Attributes/OrderAttribute.cs
using System;

namespace SysAudit.Attributes
{
    [AttributeUsage(AttributeTargets.Property, Inherited = false, AllowMultiple = false)]
    public sealed class OrderAttribute : Attribute
    {
        public int Order { get; }

        public OrderAttribute(int order = 0)
        {
            this.Order = order;
        }
    }
}